CHANGELOG.md

```diff
@@ -61,7 +61,7 @@ Here is an overview of all new **experimental** features:
 ### Improvements
 
 - **General**: Add parameter queryParameters to prometheus-scaler ([#4962](https://github.com/kedacore/keda/issues/4962))
-- **General**: TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- **General**: Improve the implementation of OpenTelemetry integration ([#5101](https://github.com/kedacore/keda/issues/5101))
```

Comment:
We may want to explain what we did rather than the fact that _an_ improvement was made

Reply:
as this is not user facing change, we can should this to Other section